implementations/typescript/ockam/ockam_app/src/routes/service/ServiceCreate.svelte

```diff
@@ -2,7 +2,7 @@
   import { getCurrent } from "@tauri-apps/plugin-window";
   import { invoke } from "@tauri-apps/api/tauri";
 
-  let service = "/service/outlet";
+  let service = "";
```

Comment:
I originally added a default value following the "default" principles we've been implementing in the CLI, and I think it's comfortable when you are just working with a single service because I don't have to type anything. Why do you think we should remove the default value? Just curious.

Reply:
I just couldn't find a good default name. I am also not sure if this is a good idea to encourage users to share a service called "service" since this will show up in the invitee email. Their experience will be better if they receive a meaningful name

Reply:
Yeah that makes sense. Let's do that then! Thanks 🙏